Assets/AppCenter/Editor/AppCenterSettingsMakerIos.cs

```diff
@@ -24,7 +24,8 @@ public class AppCenterSettingsMakerIos : IAppCenterSettingsMaker
     private const string InstallUrlToken = "APPCENTER_UNITY_USE_CUSTOM_INSTALL_URL";
     private const string UseCustomMaxStorageSize = "APPCENTER_USE_CUSTOM_MAX_STORAGE_SIZE";
     private const string MaxStorageSize = "APPCENTER_MAX_STORAGE_SIZE";
-
+    private const string UpdateTrackSearchText = "1/*UPDATE_TRACK*/";
```

Comment:
There is a space in the objective c file, I am confused.

Reply:
Fixed

Reply:
Interesting, do other settings work? The ones that use the same pattern.